Cargo.toml

```diff
@@ -14,7 +12,9 @@ members = [
 ]
 
 exclude = [
+  "partiql-conformance-test-generator",
```

Comment:
Moving the packages out of the workspace results in the following error in the GitHub Actions for the "Cargo Test of the conformance tests (can fail) and save to json file" step

```
error: cannot specify features for packages outside of workspace
```

Reply:
Fair, this is a non-blocker so can move it back for now.